reference/dic_tags.rst

```diff
@@ -96,9 +93,9 @@ services:
         use App\Lock\PostgresqlLock;
         use App\Lock\SqliteLock;
 
-        $container->register('app.mysql_lock', MysqlLock::class)->setPublic(false);
-        $container->register('app.postgresql_lock', PostgresqlLock::class)->setPublic(false);
-        $container->register('app.sqlite_lock', SqliteLock::class)->setPublic(false);
+        $container->register('app.mysql_lock', MysqlLock::class);
```

Comment:
Looks like a left over here, we should use the `ContainerConfigurator` here. Would you mind changing it too, or we could do it in another PR.